cmd/migrate/README.md

```diff
@@ -7,7 +7,7 @@
 [Release Downloads](https://github.com/golang-migrate/migrate/releases)
 
 ```bash
-$ curl -L https://github.com/golang-migrate/migrate/releases/download/$version/migrate.$platform-amd64.tar.gz | tar xvz
+$ curl -L https://github.com/golang-migrate/migrate/releases/download/$version/migrate.$platform.tar.gz | tar xvz
```

Comment:
Could you update this to `migrate.$os-$arch.tar.gz` so we're consistent with Go's [terminology](https://go.dev/doc/install/source#environment)?